### PR TITLE
Bump setup instructions - ruby setup steps + nvm setup steps

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

Stemming from a discussion in https://github.com/DCAFEngineering/dcaf_case_management/pull/2299#discussion_r676034512 this PR adjusts setup instructions a bit to accommodate installing from nvm. (I have this going into main but can also punch it into @xmunoz 's working branch if that's easier.)

This pull request makes the following changes:
* setup instructions bump for ruby and nvm
* allowlists `.ruby-version`
* checks in `.nvmrc`

It relates to the following issue #s: 
* stems from #2299

For reviewer:
* Adjust the title to explain what it does for the notification email to the listserv.
* Tag this PR:
  * `feature` if it contains a feature, fix, or similar. This is anything that contains a user-facing fix in some way, such as frontend changes, alterations to backend behavior, or bug fixes.
  * `dependencies` if it contains library upgrades or similar. This is anything that upgrades any dependency, such as a Gemfile update or npm package upgrade.
* If it contains neither, no need to tag this PR.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.7.2
```
Comment (xmunoz) [inline .ruby-version:1]:
```suggestion
2.7.4
```

**Comment by xmunoz** [comment]
Just changed the base. I think it makes sense to merge this into my PR since it's very much linked.

File: Gemfile.lock [hunk 2]
```diff
@@ -84,9 +84,9 @@ GEM
       actionpack (>= 5.2)
```
[the remark this answers is not in the document]
Comment (colinxfleming) [reply Gemfile.lock:84]: yeah, I think is prob out of sync with main. I'll rebase.

**Review by xmunoz — APPROVED** [review]

**Comment by colinxfleming** [comment]
into the branch it goes! thx @xmunoz

**Pull request merged**